dotnet/MomentoExamples/MomentoApplicationPresignedUrl/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Text;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using MomentoSdk;
+
+namespace MomentoApplicationPresignedUrl
+{
+    class Program
+    {        
+        private static readonly string SIGNING_KEY = Environment.GetEnvironmentVariable("SIGNING_KEY");
+        private static readonly string ENDPOINT = Environment.GetEnvironmentVariable("ENDPOINT");
+        private static readonly uint URL_TTL_MINUTES = 10;
+        private static readonly string CACHE_NAME = "cache";
+        private static readonly string OBJECT_KEY = "MyKey";
+        private static readonly string OBJECT_VALUE = "MyData";
+        private static readonly uint OBJECT_TTL_SECONDS = 60 * 60 * 24 * 14; // 14 days
+
+        private static readonly HttpClient client = new HttpClient();
+
+        private static string Base64Encode(string plainText) {
+            var plainTextBytes = System.Text.Encoding.UTF8.GetBytes(plainText);
+            return System.Convert.ToBase64String(plainTextBytes);
+        }
+
+        private static async Task RunPresignedUrlExample(Uri setUri, Uri getUri)
+        {
+            Console.WriteLine($"Posting value with signed URL for {CacheOperation.SET}: {OBJECT_VALUE}");
+            var json = JsonConvert.SerializeObject(Base64Encode(OBJECT_VALUE));
```

Comment:
Add //TODO here to highlight why base64 is needed for now, but we intend to fix it later?